scvi/inference/variational_inference.py

```diff
@@ -67,14 +67,27 @@ def ll(self, name, verbose=False):
 
     ll.mode = 'min'
 
-    def imputation_errors(self, name, **kwargs):
-        return imputation(self.model, self.data_loaders[name], **kwargs)
+    def imputation(self, name, verbose=False, rate=0.1, n_samples=1, n_epochs=1, corruption="uniform",
+                   title="Imputation"):
+        original_list, imputed_list = imputation(self, name, rate=rate, n_epochs=n_epochs, n_samples=n_samples,
+                                                 corruption=corruption)
+        # Median of medians for all distances
+        median_imputation_score = np.median(np.abs(np.concatenate(original_list) - np.concatenate(imputed_list)))
+
+        # Mean of medians for each cell
```

Comment:
Is this what you had in mind @jeff-regier ?

Reply:
Yes, I think that is good since that's what in the paper, even though it's kind of an arbitrary metric.